src/callbacks/callbacks.jl

```diff
@@ -407,16 +394,17 @@ function save_to_disk_func(integrator)
     end
 
     if !isnothing(p.atmos.vert_diff)
-        (; ρ_dif_flux_uₕ, ρ_dif_flux_h_tot, ρ_dif_flux_q_tot) = p
+        (; ρ_flux_uₕ, ρ_flux_h_tot) = p.sfc_conditions
         vert_diff_diagnostic = (;
-            sfc_flux_momentum = ρ_dif_flux_uₕ ./ Spaces.level(Y.c.ρ, 1),
-            sfc_flux_energy = ρ_dif_flux_h_tot,
+            sfc_flux_momentum = ρ_flux_uₕ ./
```

Comment:
Before we were dividing by `Spaces.level(Y.c.ρ, 1)` Was that a bug?

Reply:
I think it was, yes.